src/asmcnc/comms/foreign_dictionary.txt

```diff
@@ -709,12 +709,31 @@ SmartBench has paused the job, to prevent further damage.	SmartBench hat den Auf
 Before raising the Z axis, ensure that you release any side-load on the tool by jogging the X and Y axis a very small amount (e.g. 0.5mm), away from the side of the material.	Bevor Sie die Z-Achse anheben, vergewissern Sie sich, dass Sie das Werkzeug von der seitlichen Belastung befreien, indem Sie die X- und Y-Achse ein kleines Stück (z. B. 0,5 mm) von der Seite des Materials wegbewegen.	Avant d’élever l’axe Z, assurez-vous d’avoir relâcher toute charge latérale sur l’outil en l’éloignant légèrement sur les axes X et Y (par exemple 0.5mm).	Prima di sollevare l'asse Z, assicurarsi di scaricare ogni frizione laterale dall'utensile muovendo gli assi X e Y di pochissimo (ad esempio 0,5 mm), allontanandosi dal bordo del materiale.	Varmista ennen Z-akselin nostamista, että vapautat työkaluun kohdistuvan sivuttaiskuorman käsin ajamalla X- ja Y-akselia hyvin pienen määrän (esim. 0,5 mm) poispäin materiaalin sivusta.	Voordat u de Z-as omhoog brengt, moet u ervoor zorgen dat de zijwaartse druk op het gereedschap wordt weggenomen door de X- en Y-as een heel klein beetje (bijv. 0,5 mm) van de zijkant van het materiaal af te bewegen.	Przed podniesieniem osi Z, upewnij się czy usunięto jakiekolwiek obciążanie boczne narzędzia - w tym celu przesuń głowicę w osiach X i Y o bardzo małą wartość (np. 0,5mm) z dala od materiału.	Før du hæver Z-aksen, skal du sikre dig, at du frigør enhver sidebelastning på værktøjet ved at rykke X- og Y-aksen en lille smule (f.eks. 0,5 mm) væk fra siden af materialet.
 Enabled	Aktiviert	activé	Abilitato	Aktivert	Geactiveerd	Włączony	Aktiveret	활성화	Aktiverad	有効	Activado
 Disabled	Deaktiviert	désactivé	Disabilitato	Deaktivert	Gedeactiveerd	Wyłączony	Deaktiveret	장애가 있는	Avaktiverad	無効	Desactivado
 Failed to toggle SSH Service	SSH Dienst Umschaltung fehlgeschlagen	Impossible de commuter le service SSH	Passaggio a SSH service fallito	Kan ikke veksle SSH-tjeneste	SSH service herstart mislukt	Nie udało się przełączyć usługi SSH	Kunne ikke slå SSH-service til/fra	SSH 서비스를 전환하지 못했습니다	Misslyckade att växla till SSH service	SSHサービスの切り替えに失敗しました	Error al activar el servicio SSH
 Reboot console and try again	Starten Sie Ihre Konsole neu und versuchen Sie es erneut	Redémarrez votre console et réessayez	Riavvia la console e prova ancora	Restart konsollen og prøv på nytt	Herstart de console en probeer het opnieuw	Zrestartuj interfejs i spróbuj ponownie	Genstart din konsol og forsøg igen	콘솔을 재부팅하고 다시 시도하십시오	Starta om din manöverpanel och försök igen	コンソールを再起動し、再度お試しください。	Reinicie la consola e inténtelo de nuevo
 SSH	SSH	SSH	Shell Sicura	SSH	SSH	SSH	SSH
+Upgrade	Upgrade	mise à niveau	Aggiornamento	Päivitä	Upgrade	Modernizacja	Opgrader
```

Comment:
Ensure localisation spreadsheet colours are updated to reflect additions here :)